javascript/javascript2/week1/findthedanishletters.js

```diff
@@ -0,0 +1,17 @@
+function findDanishLetter(danishString) {
+    const dasnishLetters = ['æ', 'ø', 'å'];
+
+    danishString = danishString.toLowerCase();
+    var countÅ = danishString.split('å').length - 1;
+    var countØ = danishString.split('ø').length - 1;
+    var countÆ = danishString.split('æ').length - 1;
```

Comment:
The most up-to-date convention around javaScript variables says that we should avoid using **var** like the pest😬. We use **const** or **let** instead.